src/components/diagrams/networkAreaDiagram/network-area-diagram-content.jsx

```diff
@@ -177,15 +176,13 @@ function NetworkAreaDiagramContent(props) {
             }
 
             // Repositioning the previously moved nodes
-            if (!props.loadingState) {
-                const correspondingMovements = nadNodeMovements.filter(
-                    (movement) => movement.nadIdentifier === nadIdentifier
-                );
-                if (correspondingMovements.length > 0) {
-                    correspondingMovements.forEach((movement) => {
-                        diagramViewer.moveNodeToCoordinates(movement.equipmentId, movement.x, movement.y);
-                    });
-                }
+            const correspondingMovements = nadNodeMovements.filter(
```

Comment:
if removed `props.loadingState` check here,
is it still a `useLayoutEffect` dependency ?

Reply:
Good catch. It was not needed.